aiocache/__init__.py

```diff
@@ -8,7 +8,7 @@
 
 logger = logging.getLogger(__name__)
 
-AIOCACHE_CACHES: Dict[str, Type[BaseCache[Any]]] = {SimpleMemoryCache.NAME: SimpleMemoryCache}
+_AIOCACHE_CACHES: Dict[str, Type[BaseCache[Any]]] = {SimpleMemoryCache.NAME: SimpleMemoryCache}
```

Comment:
I guess the only purpose of this is to load them into `__all__` now.
In which case, the SimpleMemory Cache can probably be put in directly, and this can just be initialised to an empty list (there's no point of a dict anymore).

Reply:
Fixed.  https://github.com/aio-libs/aiocache/pull/948/commits/a9c7259475da57a307935c62cf8c90dde9534eef